web-ui/src/main/resources/catalog/components/metadataactions/MetadataActionsDirective.js

```diff
@@ -470,9 +473,14 @@
                 } else {
                   scope.userGroupDefined = false;
                 }
-              });
+              }).finally(function() {
+                scope.groupsLoaded = true;
+          });
 
           scope.save = function() {
+            if (!scope.selectedUserGroup) {
+              return
```

Comment:
I know is valid without the semicolon, but I think we use the semicolon to end the lines in the current code, please check to update it.

Reply:
I fixed your comment Jose.